#goals
float64[] x_values
float64[] y_values
float64[] phi_values
---
#results
bool goal_reached
---
#feedback
int32 fdbk